channels/stable

```diff
@@ -58,7 +58,7 @@ spec:
     #requiredVersion: 1.10.0
     kubernetesVersion: 1.10.3
   - range: ">=1.9.0-alpha.1"
-    recommendedVersion: 1.9.0
+    recommendedVersion: 1.9.2
```

Comment:
this is a pita to maintain. if i threw a some code together that selected the latest patch version of a given release, would that be crazy or overkill? i think this > < thing is not ideal.